### PR TITLE
Add Perl 6

### PR DESCRIPTION
**Pull request by sfischer13**
**https://github.com/sfischer13/awesome-perl6**

**Perl 6 programming language, which is not the same as Perl**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

File: readme.md [hunk 1]
```diff
@@ -170,6 +170,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [Perl 6](https://github.com/sfischer13/awesome-perl6) - Newest member of the Perl family.
```
Comment (ErikBoesen) [inline readme.md]: I don't feel this should assume knowledge of what Perl is. Perhaps something like "A new standard for Perl, a high-level interpreted scripting language" could be better.

Comment (sfischer13) [reply readme.md]: Thanks for the feedback and sorry for the delay. I adjusted the description.

**Comment by sfischer13** [comment]
I applied the suggestion and added comments to #1096 and #1393. @sindresorhus, are there further suggestions?

**Comment by sfischer13** [comment]
@sindresorhus, is there anything else I should do?
I applied the suggestions from @ErikBoesen and reviewed #1096,  #1393 and #1461.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

**Comment by sindresorhus** [comment]
> * Includes a project logo/illustration whenever possible.

File: readme.md [hunk 2]
```diff
@@ -177,6 +177,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3#readme) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta#readme) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris#readme) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [Perl 6](https://github.com/sfischer13/awesome-perl6) - New standard for Perl, a multi-paradigm language with exceptional string-processing support.
```
Comment (sindresorhus) [inline readme.md]: I think it should be added below the `Perl` entry.

**Comment by sindresorhus** [comment]
Otherwise looks very good :)

**Comment by sfischer13** [comment]
Thanks a lot for your feedback! I am working on it.

**Comment by sfischer13** [comment]
- [ ] Can you fix the merge conflict?
- [ ] Includes a succinct description of the project/theme at the top of the readme.
- [ ] Includes a project logo/illustration whenever possible.
- [ ] I think it should be added below the `Perl` entry.

**Comment by sindresorhus** [comment]
> Includes a project logo/illustration whenever possible.

Ping

**Comment by sfischer13** [comment]
Sorry for the delay.

I guess I won't add the Perl 6 logo. Its usage seems to be discouraged ([camelia.txt](https://raw.githubusercontent.com/perl6/mu/master/misc/camelia.txt)) for community projects.

> Camelia is intended primarily to represent Perl 6, The Language,
not any other aspect of Perl 6 culture, corporate or otherwise.

**Pull request closed without merging**